src/app/header/components/TermButtons.tsx

```diff
@@ -4,20 +4,20 @@ import React, { useContext, useEffect, useState } from 'react';
 import { TermContext } from '../../context/TermContext';
 import { getCurrentTerms, getReadableTerm } from '../../shared/utils/terms';
 
+const terms = getCurrentTerms();
```

Comment:
Why is this moved to the global scope? Shouldn't just removing `terms` from the deps fix the issue?

Reply:
either works, is there a preferred way?

Reply:
Well if it doesn't have to be global it probably shouldn't be

Reply:
I'll just throw the argument out there for it being global: there's a linting rule which wants terms in the dependency array when it's scoped to the function so technically the dependency array will be incomplete. AFAIK both ways fix the bug but both also have "cons".

Reply:
ahh ok thanks, global then